observationtree/src/main/scala/explore/observationtree/TargetObsQueries.scala

```diff
@@ -60,23 +63,7 @@ object TargetObsQueries {
       for {
         target  <- c.as[SiderealTarget]
         obsJson <- c.downField("observations").as[List[Json]].map(_.map(_.hcursor))
-        obsList <- Traverse[List].traverse(obsJson) { obs =>
-                     for {
-                       id              <- obs.downField("id").as[ExploreObservation.Id]
-                       status          <- obs.downField("status").as[ObsStatus]
-                       conf            <- obs.downField("configuration").as[String]
-                       constraintsName <- obs.downField("constraint").downField("name").as[String]
-                       duration        <-
-                         obs.downField("duration_seconds").as[Long].map(Duration.ofSeconds)
-                     } yield ExploreObservation(
-                       id,
-                       target,
-                       status,
-                       conf,
-                       constraintsName,
-                       duration
-                     )
-                   }
+        obsList <- Traverse[List].traverse(obsJson)(obsDecoder(target).apply)
```

Comment:
I think this can be written as `obsJson.traverse(obsDecoder...`

Reply:
Yes, that's simpler. Thank you.